variables.tf

```diff
@@ -1,76 +1,73 @@
-
+# KMS Key
 variable "description" {
-  default     = null
   description = "The description of the key as viewed in AWS console."
   type        = string
+  default     = null
 }
 
 variable "key_usage" {
-  default     = "ENCRYPT_DECRYPT"
   description = "Specifies the intended use of the key. Valid values: ENCRYPT_DECRYPT or SIGN_VERIFY. Defaults to ENCRYPT_DECRYPT"
   type        = string
+  default     = "ENCRYPT_DECRYPT"
 }
 
 variable "customer_master_key_spec" {
-  default     = "SYMMETRIC_DEFAULT"
   description = "Specifies whether the key contains a symmetric key or an asymmetric key pair and the encryption algorithms or signing algorithms that the key supports. Valid values: SYMMETRIC_DEFAULT, RSA_2048, RSA_3072, RSA_4096, ECC_NIST_P256, ECC_NIST_P384, ECC_NIST_P521, or ECC_SECG_P256K1. Defaults to SYMMETRIC_DEFAULT. "
   type        = string
+  default     = "SYMMETRIC_DEFAULT"
 }
 
-variable "policy" {
-  default     = null
+variable "kms_policy" {
   description = "A valid policy JSON document. Although this is a key policy, not an IAM policy, an aws_iam_policy_document, in the form that designates a principal, can be used"
   type        = string
+  default     = null
 }
 
 variable "bypass_policy_lockout_safety_check" {
-  default     = false
   description = "A flag to indicate whether to bypass the key policy lockout safety check. Setting this value to true increases the risk that the KMS key becomes unmanageable. Do not set this value to true indiscriminately.The default value is false. "
   type        = bool
+  default     = false
 }
 
 variable "deletion_window_in_days" {
-  default     = 30
   description = " The waiting period, specified in number of days. After the waiting period ends, AWS KMS deletes the KMS key. If you specify a value, it must be between 7 and 30, inclusive. If you do not specify a value, it defaults to 30. If the KMS key is a multi-Region primary key with replicas, the waiting period begins when the last of its replica keys is deleted. Otherwise, the waiting period begins immediately."
   type        = number
+  default     = 30
 }
```

Comment:
Although the default here is `30`, I would make the default for this module to be 7 days, because of the many tests we are doing. This way it would take less days to delete a key compared to having the key delete after 30 days. What do you think @ndegwajohn